.circleci/config.yml

```
@@ -366,6 +367,15 @@ jobs:
           min_exit_period: 20
       - snapshot:
           min_exit_period: 120
+      - run: 
+          name: Gather all the snapshots and publish a release
+          command: |
+            if [ "$CIRCLE_BRANCH" = "master" ]; then
+              DATA=$(cat /tmp/release | jq -Rs '.')
+              curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST -d "{\"tag_name\": \"v1.0.0\",\"body\": ${DATA} }" "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases"
```

```suggestion
              curl -H "Authorization: token ${GITHUB_TOKEN}" -X POST -d "{\"tag_name\": \"${TAG}.plasma-${CONTRACT_SHA}\",\"body\": ${DATA} }" "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases"
```

> any idea how to bump the tag when creating a release?

@InoMurko Something like this?

---

Oh, this is great 👍🏻